src/test/java/ca/concordia/soen6441/project/phases/IssueOrderTest.java

```diff
@@ -210,4 +210,38 @@ void testAdvance_Error_NotANeighbor() {
         verify(d_player, never()).issue_order(any(Advance.class));
         assertTrue(d_outContent.toString().contains("ERROR: Country2 is not a neighbor"));
     }
+    /**
+     * Tests airlift when the source and target countries are the same.
+     * Ensures that no order is issued.
+     */
+    @Test
+    void testAirlift_SourceAndTargetSame_NoOrderIssued() {

```

Comment:
Good, but you should include at least three more testcases:
- When source isnt owned by you
- When dest isnt owned by you
- Golden Case: when everything is correct and the issueOrder goesthrough